src/tests/components/simulation/ConsumptionEmailResult.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { mount } from '@vue/test-utils'
+import ConsumptionEmailResult from '@/components/simulation/ConsumptionEmailResult.vue'
+import CheckBox from '@/components/simulation/CheckBox.vue'
+import { legalList } from '@/constants/legalLinks'
+
+describe('ConsumptionEmailResult', () => {
+  it('renders correctly', () => {
+    const wrapper = mount(ConsumptionEmailResult)
+
+    expect(wrapper.html()).toMatchSnapshot()
```

Comment:
What is toMatchSnapshot used for ? If the component is modified, does this comparison evolve and/or become false ?